_studio/shared/umc/codec/h265_dec/src/umc_h265_task_supplier.cpp

```diff
@@ -1662,6 +1677,13 @@ UMC::Status TaskSupplier_H265::ProcessNalUnit(UMC::MediaDataEx *nalUnit)
     case NAL_UT_SPS:
     case NAL_UT_PPS:
         umcRes = DecodeHeaders(nalUnit);
+#if (MFX_VERSION >= 1025)
+        UMC::MediaData::AuxInfo* aux = (nalUnit) ? nalUnit->GetAuxInfo(MFX_EXTBUFF_DECODE_ERROR_REPORT) : NULL;
```

Comment:
CI is complaining on initialization inside case, to fix that just introduce new block:
```
{
    #if (MFX_VERSION >= 1025)
        UMC::MediaData::AuxInfo* aux = (nalUnit) ? nalUnit->GetAuxInfo(MFX_EXTBUFF_DECODE_ERROR_REPORT) : NULL;
...
}
```

Reply:
Added the block in new patch.